### ISSUE TITLE
ZeroDivisionError  when running select_branch_cells

### ISSUE THREAD

**Issue by jam8924** [dpeerlab/Palantir#152]
I've been following the Palantir tutorial (https://github.com/dpeerlab/Palantir/blob/master/notebooks/Palantir_sample_notebook.ipynb) and everything has been working great on my data until I try to mask cells using:

masks = palantir.presults.select_branch_cells(proteomics, q=.01, eps=.01)

No matter what parameters I change I get the error:

```
Traceback (most recent call last):
  File "psudoprot-v3.py", line 138, in <module>
    masks = palantir.presults.select_branch_cells(proteomics, q=.9, eps=.001)
  File "/home/jmelendez/.local/lib/python3.8/site-packages/palantir/presults.py", line 555, in select_branch_cells
    nsteps = n // step
ZeroDivisionError: integer division or modulo by zero
```
I saw in the documentation notes for Version 1.3.4rc "avoid devision by zero in select_branch_cells for very small datasets." 

My data set is particularly small (~240 cells). Is there a work around for this error? What can I do?

**Comment by katosh** [comment]
Hi @jam8924, thank you for reporting.

This was fixed by 12e363be03bf2d3e285ce0edaf588249faf5d847 and can be installed with
```
pip install --ignore-installed --no-deps 'git+https://github.com/dpeerlab/Palantir'
```
Alternatively, you can set `palantir.presults.PSEUDOTIME_RES = 240` to avoid this error in the older version.

**Comment by katosh** [comment]
We just released Palantir v1.3.4, so, you can now
```
pip install palantir --upgrade
```
not install the patched version.
